blog/rome-incentives/README.md

```diff
@@ -0,0 +1,237 @@
+
+Rome is set to be launched on Monday 16th March. Read more about the procedure for the release [here](https://blog.joystream.org/from-acropolis-to-rome/). Updated guides for testnet roles will be released on Sunday 15th March. Prospective participants should familiarize themselves with these guides before launch for the best chance of winning a spot in one of these competitive roles.
+
+Executive Summary
+-----------------
+
+- `Content Creators` receive up to $0.50 per qualifying item of content uploaded plus bonuses
+- `Content Curators` earn either $15 or $30 per week, depending on workload
+- `Storage Providers` will share $120 per week, plus $0.03/GB per week
+- `Validators` are competing for their share of $50 per week
+- `Council Members` are paid $10 for getting elected, with a $10 bonus
+- `Bug Reporters` can continue to earn monero for reporting and/or fixing bugs
+
+Who, Why, What and How?
+-----------------------
+
+As with previous incentivized testnets, anyone who wants to can participate on Rome and earn monero, but there is some extra help and guidance for those that ask for it in our [Telegram group](https://t.me/JoyStreamOfficial). 
+
+The philosophy behind why was discussed in detail in a [previous post](https://blog.joystream.org/pay-for-play/). In short, we are paying to prepare active participants for an autonomous platform and test the incentive structure with "real money".
+
+What you can do to qualify for getting paid on Rome is described below. More information, guides on how to get setup and extra support can be found in our helpdesk repo on GitHub.
+
+Participants should also note that some important changes to the faucet will take place upon the launch of Rome. To prevent Sybil attacks, tokens will be made available primarily through our communication channels (including Telegram, Twitter and our mailing list). The faucet will continue to operate, but will offer considerably lower payouts.
+
+Paid Roles
+----------
+
+- Content Creator
+- Content Curator
+- Storage Provider
+- Validator
+- Council Member
+- Bug Reporter
+
+If you find one or more of these roles interesting, or want to learn about future roles on the network, we encourage you to visit the new [roles](https://joystream.org/roles) section on our website. Here you can browse through them and get a sense of what they will mean to the network, along with responsibilities and requirements.
+
+You can also sign up for role specific newsletters that will keep you updated on development and incentives!
+
+Content Creators
```

Comment:
This seems a little overwhelming.

Maybe it can be a little more concise? If you don't mind, I'll make a suggestion?